### PR TITLE
AGS: Updates Steam "Beer!" & various free titles

### PR DESCRIPTION
**Pull request by MarcoBorrini99**
This PR updates Steam version for "Beer!" plus about 25 entries from https://www.adventuregamestudio.co.uk/

### CODE REVIEW

File: engines/ags/detection_tables.h [hunk 1]
```diff
@@ -1834,6 +1834,7 @@ const PlainGameDescriptor GAME_NAMES[] = {
 	{ "lemasabachthani", "Lema Sabachthani" },
 	{ "leogravedigger", "Leo the Gravedigger's Adventure" },
 	{ "leopoldkettle", "The Surprisingly Short Adventure of Leopold Kettle" },
+	{ "lesangimmortels", "Les Sang de Immortels" },
```
Comment (tag2015) [inline engines/ags/detection_tables.h:1837]:
```suggestion
	{ "lesangimmortels", "Le Sang des Immortels" },
```

File: engines/ags/detection_tables.h [hunk 2]
```diff
@@ -8723,6 +8740,7 @@ const AGSGameDescription GAME_DESCRIPTIONS[] = {
 	GAME_ENTRY_EN("thedwarvendaggerofblitz", "teste2.exe", "06a03fe35791b0578068ab1873455463", 9874658),
 	GAME_ENTRY_EN("thedwarvendaggerofblitzch2", "TDDOB.exe", "82da2565c456dcfb265ded6fe3189c0b", 69210852),
 	GAME_ENTRY_EN("theelevator", "Elevator.exe", "f10516e88ec858700804ee69d041aead", 29506915),  // Windows
+	GAME_ENTRY_EN("theelevator", "Elevator.exe", "3b7cceb3e4bdb031dc5d8f290936e94b", 6563864),  // Windows
```
Comment (tag2015) [inline engines/ags/detection_tables.h:8743]: As you can see in commit 27da99f608e33ce9683505229998aa447473e9f2, there was a mixup between "The Elevator" and "The Elevator: Rise from the Abyss". This checksum refers to the OROW version of `elevatorriseabyss`, which is already present

Comment (MarcoBorrini99) [reply engines/ags/detection_tables.h]: Fix committed. 

Now:

- The Elevator - Rise from the Abyss! (English)
- The Elevator - Rise from the Abyss! (OROW⁄English)
- The Elevator (English)

are correctly recognized

**Comment by tag2015** [comment]
Thanks! Squashing and merging

**Pull request merged**